controller_manager/test/test_controller_manager.cpp

```diff
@@ -389,10 +389,8 @@ TEST_P(TestControllerManagerWithUpdateRates, per_controller_equal_and_higher_upd
     // In case of a non perfect divisor, the update period should respect the rule
     // [cm_update_rate, 2*cm_update_rate)
     EXPECT_THAT(
-      test_controller->update_period_,
-      testing::AllOf(
-        testing::Ge(rclcpp::Duration::from_seconds(1.0 / cm_update_rate)),
-        testing::Lt(rclcpp::Duration::from_seconds(2.0 / cm_update_rate))));
+      test_controller->update_period_.seconds(),
+      testing::AllOf(testing::Ge(0.9 / cm_update_rate), testing::Lt((1.1 / cm_update_rate))));
```

Comment:
Not complaining, but is there a reason for having >=0.9 , but <1.1?

Reply:
This is because the CI doesn't have the real-time kernel. In order to avoid flaky tests, I had to choose these ranges. I hope that's okay.

Reply:
I was more referring to "greater **or equal**" vs "less **than**"

Reply:
Ahh sure, it can be modified to "greater than. You are right. Thanks for the feedback